dbt/include/snowflake/macros/adapters.sql

```diff
@@ -78,9 +78,10 @@
   {%- set sql_header = config.get('sql_header', none) -%}
 
   {{ sql_header if sql_header is not none }}
+  -- CHECK ORDER
   create or replace {% if secure -%}
     secure
-  {%- endif %} view {{ relation }}
+  {%- endif %} temporary view {{ relation }}
```

Comment:
Alternatives considered include:
1. forcing all `create_view_as` functions to take a `temporary` boolean and guard against that here.
2. duplicate this method with temporary active and call that in the incremental materialization
    * really only helps if the goal is to keep `create_view_as` unadulterated, but I saw no real need for that